apps/site-nft-landing/src/styles.less

```diff
@@ -352,7 +377,6 @@ p.smaller {
     }
     &:hover {
       background: linear-gradient(0deg, rgba(255, 255, 255, 0.3), rgba(255, 255, 255, 0.2)), linear-gradient(132.82deg, rgba(255, 255, 255, 0.2) 25.95%, rgba(255, 255, 255, 0) 135.13%);
-      filter    : drop-shadow(0px 100px 80px rgba(0, 0, 0, 0.02)) drop-shadow(0px 64.8148px 46.8519px rgba(0, 0, 0, 0.0151852)) drop-shadow(0px 38.5185px 25.4815px rgba(0, 0, 0, 0.0121481)) drop-shadow(0px 20px 13px rgba(0, 0, 0, 0.01)) drop-shadow(0px 8.14815px 6.51852px rgba(0, 0, 0, 0.00785185)) drop-shadow(0px 1.85185px 3.14815px rgba(0, 0, 0, 0.00481481));
```

Comment:
The dropshadow here was causing a shimmering background on firefox, it did virtually nothing for the image so I just removed it.